src/plugins/newGuildSettings/index.tsx

```diff
@@ -68,12 +72,50 @@ const settings = definePluginSettings({
     }
 });
 
+const guildPopoutPatch: NavContextMenuPatchCallback = (children, { guild }: { guild: Guild, onClose(): void; }) => {
+    if (!guild) return;
+
+    children.push(
+        <Menu.MenuItem
+            label="Apply NewGuildSettings"
+            id="vc-newguildsettings-apply"
+            icon={CogWheel}
+            action={() => handleSettings(guild.id)}
+        />
+    );
+};
+
+function handleSettings(guildId: string | null) {
+    if (guildId === "@me" || guildId === "null" || guildId == null) return;
+    updateGuildNotificationSettings(guildId,
+        {
+            muted: settings.store.guild,
+            suppress_everyone: settings.store.everyone,
+            suppress_roles: settings.store.role,
+            mute_scheduled_events: settings.store.events,
+            notify_highlights: settings.store.highlights ? 1 : 0
+        });
+    if (settings.store.messages !== 3) {
+        updateGuildNotificationSettings(guildId,
+            {
+                message_notifications: settings.store.messages,
+            });
+    }
+    if (settings.store.showAllChannels && isOptInEnabledForGuild(guildId)) {
+        toggleShowAllChannels(guildId);
+    }
+}
+
+
 migratePluginSettings("NewGuildSettings", "MuteNewGuild");
 export default definePlugin({
     name: "NewGuildSettings",
     description: "Automatically mute new servers and change various other settings upon joining",
     tags: ["MuteNewGuild", "mute", "server"],
     authors: [Devs.Glitch, Devs.Nuckyz, Devs.carince, Devs.Mopi, Devs.GabiRP],
+    contextMenus: {
+        "guild-context": guildPopoutPatch,
```

Comment:
this will also add it to the guild header popout
```suggestion
        "guild-context": contextMenuPatch,
        "guild-header-popout": contextMenuPatch
```